terraform-provider-aws.yaml

```diff
@@ -22,6 +22,11 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: b35c1b3529733ca7be22c640a29eda61ba624fd2
 
+  - runs: |
+      # fix GHSA-m425-mq94-257g / CVE-2023-44487
+      go get google.golang.org/grpc@v1.58.3
+      go mod tidy
+
```

Comment:
Why don't you add it to the `go/build` as `with.deps: google.golang.org/grpc@v1.58.3`. It would be easier.

Reply:
i did not know TIL thanks! will update

Reply:
done